src/switch_joystick.c

```diff
@@ -107,24 +107,28 @@ static _GLFWmapping s_switchMapping =
     },
 };
 
+PadState pad;
+
 void _glfwInitSwitchJoysticks(void)
 {
     _GLFWjoystick* js = _glfwAllocJoystick(s_switchMapping.name, s_switchMapping.guid,
         _SWITCH_AXIS_COUNT, _SWITCH_BUTTON_COUNT, _SWITCH_HAT_COUNT);
 
+    padConfigureInput(1, HidNpadStyleSet_NpadStandard);
+    padInitializeDefault(&pad);
```

Comment:
Add a `hidInitializeTouchScreen();` call after this (forgot to mention it earlier)